src/helpers/getNavLink.test.js

```diff
@@ -0,0 +1,17 @@
+import getNavLink from './getNavLink';
+
+describe('helpers/getNavLink', () => {
+  const inputArray = ['page', 'abc123', 'section'];
```

Comment:
In these short tests it's easier to follow if the variables get defined in every test case.